### PR TITLE
chore: update token-distribution to use our linting packages

### PR DESCRIPTION
**Pull request by tmigone**

### CODE REVIEW

**Review by Maikol — APPROVED** [review]
LGTM

File: packages/token-distribution/test/tokenLock.test.ts [hunk 1]
```diff
@@ -324,7 +325,8 @@ describe('GraphTokenLockSimple', () => {
               // Before cliff no vested tokens
               if (cliffTime.gt(0) && currentTime.lt(cliffTime)) {
                 expectedAmount = BigNumber.from(0)
-              } else {
+              }
+              else {
```
Comment (pcarranzav) [inline packages/token-distribution/test/tokenLock.test.ts:329]: This looks like a weird choice for linter output doesn't it? `} else {` looked cleaner imo, not sure if that's configurable

**Review by pcarranzav — DISMISSED** [review]
One detail but LGTM

File: packages/token-distribution/test/tokenLock.test.ts (hunk 1 above)
Comment (tmigone) [reply packages/token-distribution/test/tokenLock.test.ts:329]: yeah i actually agree with you, going to look into it, I didn't notice this.

**Review by Maikol — APPROVED** [review]

**Pull request merged**